admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -104,6 +110,50 @@ export class AlertBannerModel extends HoistModel {
         this.refreshAsync();
     }
 
+    loadPreset(preset: PlainObject) {
+        this.formModel.setValues({...preset, expires: null});
+    }
+
+    @action
+    addPreset() {
+        const {message, intent, iconName, enableClose} = this.formModel.values;
+        const dateCreated = fmtDateTime(Date.now());
```

Comment:
We just want to save the timestamp (`Date.now()`) here - that way we can display it in the user's browser timezone when we render it into the component.  

With the approach you have currently, if someone in NY saved a preset it would capture the time in NY - if I then saw that immediately in CA it would appear to be from the future!

Saving the timestamp lets us process it into a formatted date when we _load_ the saved presets - that way everyone will see the times in their expected local zone.